packages/core/src/saml-applications/routes/index.ts

```diff
@@ -11,7 +12,7 @@ import koaGuard from '#src/middleware/koa-guard.js';
 import { buildOidcClientMetadata } from '#src/oidc/utils.js';
 import { generateInternalSecret } from '#src/routes/applications/application-secret.js';
 import type { ManagementApiRouter, RouterInitArgs } from '#src/routes/types.js';
-import { ensembleSamlApplication, validateAcsUrl } from '#src/saml-applications/routes/utils.js';
+import { ensembleSamlApplication, validateAcsUrl } from '#src/saml-applications/libraries/utils.js';
```

Comment:
It's better to use relative paths within saml-application.
Additionally, consider moving these two utility methods out of the`/libraries/utils/` directory. `/libraries/utils/` is intended for use exclusively within the libraries. We could have a `/saml-applications/utils.js`

Reply:
Missed the last part. Will reorg the `utils.ts` file before merging.